index.html

```diff
@@ -1,28 +1,97 @@
 <!DOCTYPE html>
 <html lang="de">
   <head>
-    <link rel="stylesheet" href="styles.css" />
+    <link rel="stylesheet" href="css/style.css" />
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Quiz-App</title>
+    <title>Document</title>
   </head>
   <body>
-    <header class="head-container">
-      <div class="headline">
-        <h1>Quiz-App</h1>
-      </div>
+    <header class="header__container">
+      <h1 class="header__title">
+        <img class="logo" src="img/logo2.png" width="550" alt="" />
+      </h1>
     </header>
```

Comment:
2 things:

1. The BEM is a bit off; would be `header`, `header__title` and `header__logo`
2. As you don't have text inside your `h1` you should set the `alt` tag to whatever the image is displaying. Now if someone uses the page with a screen reader, the person won't know what is in the `h1`. If you set the `alt` attribute to what a sighted visitor is seeing, then it would be read aloud.